.github/workflows/splunk-ta-otel.yml

```diff
@@ -0,0 +1,65 @@
+name: splunk-ta-otel-build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - '.github/workflows/splunk-ta-otel.yml'
+      - 'packaging/technical-addon/**'
+      - 'cmd/**'
+      - 'internal/**'
+      - 'pkg/**'
+      - 'tests/**'
+      - '!**.md'
+
+concurrency:
+  group: splunk-technical-addon-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  runs-on: "ubuntu-20.04"
```

Comment:
The workflow is not valid. .github/workflows/splunk-ta-otel.yml (Line: 22, Col: 3): Unexpected value 'runs-on'